topologies/datacenter/files/MenuOptions.yaml

```diff
@@ -49,34 +54,40 @@ labconfiglets:
       - "Leaf2-BGP-Lab"
     leaf3:
       - "Leaf3-BGP-Lab"
-    leaf4:
-      - "Leaf4-BGP-Lab"
```

Comment:
Keep the `leaf4` configlet of `Leaf4-BGP-Lab` in.  This is only a partial config, the other one is the `-Full` that we don't need.